src/item.cpp

```diff
@@ -2479,7 +2482,10 @@ void item::food_info( const item *food_item, std::vector<iteminfo> &info,
 
     if( food_item->has_flag( flag_CANNIBALISM ) &&
         parts->test( iteminfo_parts::FOOD_CANNIBALISM ) ) {
-        if( !player_character.has_flag( json_flag_CANNIBAL ) ) {
+        if( !player_character.has_flag( json_flag_CANNIBAL ) &&
+            !player_character.has_flag( json_flag_PSYCHOPATH ) &&
+            !player_character.has_flag( json_flag_SAPIOVORE ) ) && 
+            !player_character.has_flag( json_flag_NUMB ) ) {
```

Comment:
If you really intend to change how numb works, you should also check if it has other penalties and update the check here accordingly, otherwise it may still not be accurate.